core/controllers/contributor_dashboard_admin.py

```diff
@@ -594,7 +595,17 @@ class ContributorDashboardAdminStatsHandler(
                 },
                 'default_value': None
             },
-            'max_days_since_last_activity': {
+            'max_days_since_start_date_for_last_activity': {
```

Comment:
Wait, is this handler used atm? You can't simply replace the new field with the old one, it's not backward compatible.

Reply:
@nikitaevg, this handler is not in the use at the moment, it will used only when the Contributor Admin Dashboard is released.

Reply:
Ok, thanks for clarifying